README.md

```diff
@@ -1 +1,22 @@
-# badminton_booking
\ No newline at end of file
+# badminton_booking
+
+
+# Setting
+1. Update `.example_env` rename to `.env`
+2. Set your line notify token
+
+
+# Setup your env
+## Install package
+```
+pipenv install
+```
+
+## Run script with virtual env
+```
+pipenv shell
+```
+
+```
+python app.py
+```
```

Comment:
# badminton_booking

## Setting

1. Update `.example_env` rename to `.env`
2. Set your line notify token

## Setup your env

### Install package

```
pipenv install
```

### Run script with virtual env

```
pipenv shell
```

```
python app.py
```